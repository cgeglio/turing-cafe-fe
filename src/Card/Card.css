article {
  border: 3px solid #028080;
  border-radius: 3px;
  color: #028080;
  background-color: white;
  font-size: .5em;
  height: 130px;
  margin: 1em;
  width: 180px;
}

.cancel-btn {
  border: 3px solid #028080;
  background-color: #028080;
  color: white;
  cursor: pointer;
  width: 80%;
}

.cancel-btn:hover {
  border: 3px solid #028080;
  background-color: white;
  color: #028080;
}
