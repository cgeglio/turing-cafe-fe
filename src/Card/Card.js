import React from 'react';
import './Card.css';

const Card = ({ reservation, addReservation }) => {
  return (
    <article>
      <h2>{reservation.name}</h2>
      <h2>{reservation.date}</h2>
      <h2>{reservation.time}</h2>
      <h2>{reservation.number}</h2>
    </article>
  )
}

export default Card;
