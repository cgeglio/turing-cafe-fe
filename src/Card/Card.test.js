import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import { shallow } from 'enzyme';

describe('Card', () => {
  let wrapper;
  let mockReservation;
  let mockAddReservation = jest.fn();

  beforeEach(() => {
    mockReservation = {
        id: 1,
        name: "Christie",
        date: "12/29",
        time: "7:00",
        number: 12
    }
    wrapper = shallow(<Card
      reservation={mockReservation}
      addReservation={mockAddReservation}
    />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });

});
