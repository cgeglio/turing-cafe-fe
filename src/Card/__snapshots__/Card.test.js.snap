// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<article>
  <h2>
    Christie
  </h2>
  <h2>
    12/29
  </h2>
  <h2>
    7:00
  </h2>
  <h2>
    12
  </h2>
</article>
`;
